packages/devtools_app/lib/src/framework/settings_dialog.dart

```diff
@@ -93,6 +93,18 @@ class SettingsDialog extends StatelessWidget {
                 gaItem: gac.wasm,
               ),
             ),
+            // TODO(https://github.com/flutter/devtools/issues/7860): Clean-up
+            // after Inspector V2 has been released.
+            if (FeatureFlags.inspectorV2)
+              Flexible(
+                child: CheckboxSetting(
+                  notifier: preferences.inspector.inspectorV2Enabled
+                      as ValueNotifier<bool?>,
+                  title: 'Enable Inspector V2',
```

Comment:
I'm wondering if we should use "V2" for the user facing language. @anderdobo do you have an opinion on how we should phrase this for users? (e.g. "Inspector V2" vs "Inspector 2.0" vs "the new Flutter Inspector")

Reply:
I recommend that we use "the mew Flutter Inspector" so that we don't get into versioning the feature going forward.

Reply:
Done!